src/css/dark-theme.css

```diff
@@ -904,3 +904,9 @@ progress[value]::-webkit-progress-bar {
 .select2-results>.select2-results__options {
     background-color: #3a3a3a !important;
 }
+
+/* MULTIPLE-SELECT */
+
+.ms-drop ul>li.hide-radio:focus, .ms-drop ul>li.hide-radio:hover {
+    background-color: var(--subtleAccent);
+}
```

Comment:
i think changes in this file are not needed anymore?

Reply:
Yes we need that fix in multiole-select for future uses.